coalib/results/Diff.py

```diff
@@ -70,6 +72,65 @@ def from_string_arrays(cls, file_array_1, file_array_2, rename=False):
         return result
 
     @classmethod
+    def from_unified_diff(cls, unified_diff, original_file):
+        """
+        Creates a `Diff` object from given unified diff.
+
+        If this Diff is applied to the original file, the corrected
+        file will the created.
+
+        :param unified_diff:  Unified diff string.
+        :param original_file: The contents of the original file
+                              (line-splitted).This is required to
+                              construct the corrected file.
```

Comment:
1. Missing space after period
2. Omit "This is required to construct...", it's redundant. If it wouldn't have been required, you wouldn't put a parameter ;)